### DISCUSSION TITLE
Select All/None feature for Boolean Filters?

### DISCUSSION THREAD

**Discussion by vesper8** [laravel/nova-issues#3784] [Q&A]
I have a Boolean filter with many values (over 15 checkboxes) and it would very useful to me if there was a "Select All" / "Select None" functionality.

Is there currently a way to implement this kind of functionality?

Thanks

**Comment by davidhemphill** [comment]
This would be an interesting feature.

**Reply by vesper8** [reply]
I think it would be handy. It would work especially great with https://github.com/optimistdigital/nova-detached-filters (which I use and love).

**Reply by ziming** [reply]
Wow this package looks kinda like the Nova Refine package but free!

Thanks for the recommendation!

**Reply by vesper8** [reply]
Never heard of Refine.. also never heard of a paid plugin for Nova.. but wow.. they want 1000$ for 1 year?? TBH nova-detached-filters just looks outright better too
